notebooks/forecasting/forecast-timeseries.ipynb

```diff
@@ -151,7 +151,25 @@
    "source": [
     "timeseries_df = raw_timeseries_df.copy()\n",
     "timeseries_df.columns = ['ds', 'y']\n",
-    "timeseries_df.head()"
+    "display(timeseries_df.to_string())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "assert pd.api.types.is_datetime64_dtype(timeseries_df.dtypes['ds'])"
```

Comment:
Would expect these asserts to pass, as `timeseries_df` is just a copy of  `raw_timeseries_df` with its columns renamed.

Reply:
Ah yes I forgot them